rust/arrow/src/ipc/gen/SparseTensor.rs

```diff
@@ -89,7 +89,7 @@ pub enum SparseTensorIndexCOOOffset {}
 
 /// ----------------------------------------------------------------------
 /// EXPERIMENTAL: Data structures for sparse tensors
-/// Coodinate (COO) format of sparse tensor index.
+/// Coordinate (COO) format of sparse tensor index.
```

Comment:
Also, I think this is generated

Reply:
I already updated `format/SparseTensor.fbs`. Is there any another file to generate this?

Reply:
No, that should be it I think.  Thanks